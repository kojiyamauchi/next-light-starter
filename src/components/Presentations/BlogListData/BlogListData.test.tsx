/*
  Blog List Data Component Unit Test.
*/

import { BlogListData } from './'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Blog List Data Component Unit Test', () => {
  it('Snap Shot Testing', () => {
    const { asFragment } = render(<BlogListData title="Snap Shot Unit Test" date="2021/01/01" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
