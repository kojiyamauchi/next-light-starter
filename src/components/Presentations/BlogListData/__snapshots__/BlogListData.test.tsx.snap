// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog List Data Component Unit Test Snap Shot Testing 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <h2
      class="title-heading"
    >
      Snap Shot Unit Test
    </h2>
    <p
      class="date"
    >
      2021/01/01
    </p>
  </div>
</DocumentFragment>
`;
